### PR TITLE
✨ feat(core): add `console`

### PR DESCRIPTION
**Pull request by jade-gjz**

### CODE REVIEW

**Review by zwtkito — COMMENTED** [review]
I'm speechless

File: packages/core/src/console.ts [hunk 1]
```diff
@@ -0,0 +1,80 @@
...
+ * @param {...any} args
+ */
+export const jLog = (...args: any[]) =>
+  __DEV__
+    ? console.log.bind(
+        console,
+        `%c[${args.length === 1 ? 'LOG' : '%s'}]`,
+        `${sharedStyle} background: seagreen;`
+      )(...args)
+    : () => {}
```
Comment (zwtkito) [inline packages/core/src/console.ts:16]: shit

File: packages/core/src/console.ts [hunk 2]
```diff
@@ -0,0 +1,80 @@
...
+  console.info.bind(
+    console,
+    `%c[${args.length === 1 ? 'INFO' : '%s'}]`,
+    `${sharedStyle} background: dodgerblue;`
+  )(...args)
+
+/**
+ * jade console.group
+ */
+export const jGroup = console.group.bind(console, '%c[%s]', `${sharedStyle} background: slateblue;`)
```
Comment (zwtkito) [inline packages/core/src/console.ts:53]: ding

File: packages/core/src/console.ts [hunk 3]
```diff
@@ -0,0 +1,80 @@
...
+
+/**
+ * jade console.warn
+ */
+export const jWarn = console.error.bind(console, '%c[WARN]', `${sharedStyle} background: tomato;`)
+
+/**
+ * jade console.error
+ */
+export const jError = console.error.bind(console, '%c[ERROR]', `${sharedStyle} background: crimson;`)
```
Comment (zwtkito) [inline packages/core/src/console.ts:37]:
```suggestion
/**
 * jade console.error
 */
export const jError = console.error.bind(console, '%c[ERROR]', `${sharedStyle} background: crimson;`)
```

File: packages/core/types/env.d.ts [hunk 4]
```diff
@@ -0,0 +1,8 @@
+/* eslint-disable no-underscore-dangle */
+/** 当前为开发环境 */
+declare const __DEV__: boolean
+/** 当前为测试环境 */
+declare const __TEST__: boolean
+/** 当前为生产环境 */
+declare const __PROD__: boolean
```
Comment (zwtkito) [inline packages/core/types/env.d.ts:7]:
```suggestion
```
shit

File: packages/core/src/console.ts [hunk 5]
```diff
@@ -0,0 +1,80 @@
...
+  __DEV__
+    ? console.log.bind(
+        console,
+        `%c[${args.length === 1 ? 'LOG' : '%s'}]`,
+        `${sharedStyle} background: seagreen;`
+      )(...args)
+    : () => {}
+
+/**
+ * jade console.log
```
Comment (jade-gjz) [inline packages/core/src/console.ts:19]:
```suggestion
 * jade console.debug
```

File: packages/core/src/console.ts [hunk 6]
```diff
@@ -0,0 +1,80 @@
...
+ */
+export const jWarn = console.error.bind(console, '%c[WARN]', `${sharedStyle} background: tomato;`)
+
+/**
+ * jade console.error
+ */
+export const jError = console.error.bind(console, '%c[ERROR]', `${sharedStyle} background: crimson;`)
+
+/**
+ * jade console.error
```
Comment (jade-gjz) [inline packages/core/src/console.ts:40]:
```suggestion
 * jade console.info
```

**Comment by jade-gjz** [comment]
cc @zwtkito

**Review by zwtkito — COMMENTED** [review]
.

**Review by zwtkito — APPROVED** [review]

**Pull request merged**